app/controllers/organisations.php

```diff
@@ -25,16 +25,28 @@ function create() {
         ));
     }
 
-    function view() {
+    function view( $org_delete = null ) {
```

Comment:
Also, it would be better if we use session for this. As anyone going to `view/1` will be shown `Remove successful` message.